merged all html files into the doc file and deleted unwanted html files

ajit-sharmaGH opened the pull request.

ajit-sharmaGH committed:
merged all html files into the doc file and deleted unwanted html files

harshitadaswani commented on CSS/avatar.css:
```diff
@@ -1,26 +1,52 @@
+    width: 8rem;
 }
 
+
+/* 
+
+
```
remove empty space

harshitadaswani commented on CSS/avatar.css:
```diff
@@ -1,26 +1,52 @@
-    border: 1px solid var(--dark-primary-text);
-    background-color: var(--primary-bg-color);
-    box-shadow: 1px 2px 2px 1px var(--dark-primary-text);
+    /* background-color: var(--primary-bg-color); */
+    box-shadow: 1px 1px 2px 2px var(--dark-primary-text);
+}
+
+.avatar-images {
+    /* background-image: url("../images/tungsten3.jpg"); */
```
remove unnecessary comments

harshitadaswani commented on CSS/base/nav.css:
```diff
@@ -1,136 +1,60 @@
-.nav-links a {
+.navlink a {
+    margin-right: 1.5rem;
+    padding: 0.3rem 0.6rem 0.3rem 0.6rem;
```
use shorthand property `padding: 0.3rem 0.6rem`

harshitadaswani commented on CSS/base/reset.css:
```diff
@@ -94,21 +94,22 @@ button {
-    width: 70%;
+    width: 90%;
     text-align: center;
     margin: 1rem 0rem 1rem 0rem;
```
use shorthand property

harshitadaswani commented on CSS/base/reset.css:
```diff
@@ -94,21 +94,22 @@ button {
     font-weight: bold;
     border: 1px solid var(--dark-primary-text);
     box-shadow: 1px 1px 2px 2px var(--dark-primary-text);
     padding: 0.5rem 1rem 0.5rem 1rem;
```
again shorthand property

harshitadaswani commented on CSS/base/reset.css:
```diff
@@ -261,7 +254,7 @@ li {
     overflow: hidden;
     width: 101% !important;
-    max-height: 35vh !important;
+    max-height: 40vh !important;
```
don't use `!important` in component library

harshitadaswani commented on main/doc.css:
```diff
@@ -0,0 +1,83 @@
+.sidebar-link li {
+    margin-top: 0.2rem;
+    color: var(--white);
+    padding: 0.5rem 2rem 0.5rem 2rem;
```
shorthand property

harshitadaswani commented on main/doc.css:
```diff
@@ -0,0 +1,83 @@
+    width: 100vw;
+    height: calc(100vh - 3.5rem);
+    overflow: auto;
+    /* display: flex;
+    flex-direction: column;
+    align-items: center; */
```
remove comments

harshitadaswani reviewed (changes requested):
- use shorthand properties
- don't use `!important` in CL
- and try to make different PRs as it is difficult to review so many files together

JantuDeb reviewed (approved):
Implement the changes @harshitadaswani pointed out other than that everything is fine

ajit-sharmaGH committed:
fixed the changes suggested through code review
-removed unwanted comments
-removed !important
-fixed code using short hand property
